src/view/components/cell/time/body.cell.time.edit.html

```diff
@@ -1,4 +1,8 @@
-<input type="time"
-		 q-grid:focus
-		 ng-blur="$view.edit.cell.commit.execute($cell)"
-		 ng-model="$view.edit.cell.value" />
\ No newline at end of file
+<div class="q-grid-editor q-grid-time" q-grid-position="td">
+	<md-input-container>
```

Comment:
Shouldn't it be in the material templates?

Reply:
Yes